api/src/main/java/com/flipkart/poseidon/api/APILegoSet.java

```diff
@@ -99,12 +99,13 @@ public Buildable getBuildable(Request request) throws LegoSetException, ElementN
             EndpointPOJO pojo = ((APIBuildable) buildable).getPojo();
             RequestContext.set(URI, pojo.getUrl());
             RequestContext.set(ENDPOINT_NAME, pojo.getName());
-        }
 
-        String name = buildable.getName();
-        if (name != null && !name.isEmpty()) {
-            poseidonRequest.setAttribute(TIMER_CONTEXT, Metrics.getRegistry().timer("poseidon.api." + name + "." + httpMethod).time());
+            String name = pojo.getName();
+            if (name != null && !name.isEmpty()) {
+                poseidonRequest.setAttribute(TIMER_CONTEXT, Metrics.getRegistry().timer("poseidon.api." + name + "." + httpMethod).time());
+            }
```

Comment:
Alignment issue here? I see buildable.getName() changed to pojo.getName() but whole block is showing as changed